src/main/notifications/index.ts

```diff
@@ -49,6 +50,10 @@ export function displayMention(title: string, body: string, channel: {id: string
         data,
     };
 
+    if (!await PermissionsManager.doPermissionRequest(webcontents.id, 'notifications', view.view.server.url.toString())) {
```

Comment:
We add a manual permission check here because notifications are sent manually by Electron.